include/llvm-dialects/TableGen/Format.h

```diff
@@ -35,6 +35,13 @@
 
 namespace llvm_dialects {
 
+namespace llvm_support_detail =
+#if !defined(LLVM_MAIN_REVISION) || LLVM_MAIN_REVISION >= 494496
+	llvm::support::detail;
+#else
+	llvm::detail;
```

Comment:
The rest of the file uses spaces, not tabs, so this should too.

Reply:
Sorry, I will fix this.